protected/react-src/src/services/httpMethods.js

```diff
@@ -55,6 +55,28 @@ export const post = async (url, data) => {
     }
 };
 
+export const postUpload = async (url, data) => {
```

Comment:
We need to make this compatible with Laravel Authentication. We need to add the headers Accept, Content-Type and Authorization